dispatcher/DAQController.py

```diff
@@ -330,6 +335,11 @@ def check_timeouts(self, detector, command = None):
         sendstop = False
         nowtime = datetime.datetime.utcnow()
 
+        #First check how often we have been timing out, if it happend to often something
+        #  bad happend and we start from scratch again
+        if self.timeout_counter >self.max_timeout:
+            self.hypervisor.tactical_nuclear_option()
```

Comment:
Probably a good idea to return after this, I don't think we want other logic running if we're invoking this.